<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MentalHeal</class>
 <widget class="QWidget" name="MentalHeal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择想预约的心理老师</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择你想预约的时间</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="time">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>请输入一个时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>简单说说你的情况</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="situation">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>361</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Return_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Send">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>预约</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入你的姓名</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="tea1">
      <property name="text">
       <string>A老师</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="tea2">
      <property name="text">
       <string>B老师</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="tea3">
      <property name="text">
       <string>C老师</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
